app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -1,9 +1,9 @@
 .gem-c-pagination {
   display: block;
-  margin-top: $gutter;
-  margin-bottom: $gutter;
-  margin-left: -$gutter-half;
-  margin-right: -$gutter-half;
+  margin-top: govuk-spacing(6);
+  margin-bottom: govuk-spacing(6);
+  margin-left: -govuk-spacing(3);
+  margin-right: -govuk-spacing(3);
```

Comment:
Could this be merged into a single line? `margin: govuk-spacing(6) -govuk-spacing(3)`?

Also, using a negative prefix on govuk-spacing like this doesn't work. I think you need to do `-(govuk-spacing(3))`. This is the result:

![Screen Shot 2019-06-10 at 13 18 20](https://user-images.githubusercontent.com/861310/59194879-5a63a580-8b82-11e9-96e2-1b1fe83e9422.png)